Trustroots/trustroots
# Remove bower

Removes Bower package-manager dependency by installing everything via `npm` and requiring scripts directly from `node_modules`.

Todo:

Do something about `/lib/*` assets (mainly [image files](https://github.com/Trustroots/trustroots/blob/d1695a416294854baf7d0dc528bd5131f73e83a5/modules/core/client/less/modules/chosen.less#L13) referenced at core-module and [JS files for Firebase](https://github.com/Trustroots/trustroots/blob/f4bd051562b112e28b4bbeb920374b7a7713dbb2/public/push-messaging-sw.js#L6-L7)) in a few JS files and CSS files.

We could either copy these over to `public` folder with Gulp or use express static-serve handle them.

#### Testing Instructions

- Start the app both in dev (`npm start`) and prod (`NODE_ENV=production npm start`) modes
- Click around and see if you see 404s in console or anything off/missing
